.circleci/config.yml

```diff
@@ -18,4 +18,4 @@ jobs:
       - run:
           name: Ansible-lint check
           command: |
-            ansible-lint -v roles/*/*/*.yaml roles/*/*/*.yml stackstorm.yml
+            ansible-lint -x 204 -v roles/*/*/*.yml stackstorm.yml
```

Comment:
```
WARNING: Couldn't open roles/*/*/*.yaml - No such file or directory
```

Some people name files `.yaml`, some `.yml`. Both formats are perfectly valid and working in Ansible.
And while I renamed `yaml` -> `yml` several times, there is a chance that someone will commit new `yaml` files in future.

So by adding this additional `yaml` rule I want to make sure we're not missing any files here.

It's not a bug, but a feature :)